### PR TITLE
Added insurance cover - NHIF or other insurance in greencard and Registration form and validation on existing identifier

### PR DESCRIPTION
**Pull request by patryllus**
insurance_greencard
NHIF_AS_IDENTIFIER
validation
and

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/hiv/hivGreenCard.html [hunk 1]
```diff
@@ -4788,27 +4839,75 @@ <h2 class="StepTitle">Patient management</h2>
...
+                                  <tr>
+                                      <td>&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;
+                                          Medical insurance cover:
+                                          <obs conceptId="159356AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                                               id="insuranceType"
+                                               answerConceptIds="1917AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,
+                                                                 1107AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,
+                                                                 5622AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                                               style="dropdown"
+                                               answerLabels="National Health Insurance Fund Member(NHIF),None,Other"
```
Comment (ojwanganto) [inline omod/src/main/webapp/resources/htmlforms/hiv/hivGreenCard.html:4880]: Should we rename this to NHIF? I don't think we need to have the 'member' in the label but we can have a discussion

**Review by ojwanganto — CHANGES_REQUESTED** [review]

File: omod/src/main/webapp/resources/htmlforms/hiv/hivGreenCard.html (hunk 1 above)
Comment (patryllus) [reply omod/src/main/webapp/resources/htmlforms/hiv/hivGreenCard.html:4880]: Yes seems logical.

**Review by njorocs — APPROVED** [review]
Looks fine

**Review by njorocs — APPROVED** [review]
Looks fine

**Review by njorocs — APPROVED** [review]

**Review by njorocs — APPROVED** [review]

**Review by njorocs — APPROVED** [review]

**Review by njorocs — APPROVED** [review]

**Pull request merged**